source/common/http/utility.h

```diff
@@ -10,188 +10,181 @@
 #include "envoy/http/codes.h"
 #include "envoy/http/filter.h"
 #include "envoy/http/message.h"
+#include "envoy/http/query_params.h"
 
 #include "common/json/json_loader.h"
 
 #include "absl/strings/string_view.h"
 
 namespace Envoy {
 namespace Http {
+namespace Utility {
```

Comment:
Note: I changed this from a class with all static methods into a namespace in order to be able to extract QueryParams into type that was accessible from interfaces in include/...

Textually it looks like this whole file changed but that's just because it lost 2 chars of indent and a lot of 'static' keywords. Really it just lost QueryParams, which is now in include/envoy/http/query_params.h

Reply:
Thanks for the heads up, this is useful.